### PR TITLE
Adding to Reproduction Log (#2197)

### PR DESCRIPTION
**Pull request by TheFJcurve**
The reproduction process was seamless.

Ubuntu 22.04.3 LTS, 64-bit machine. Intel i7 10th Generation and NVIDIA Graphics, 16GB ram. 

Java --version: 11.0.20.1
mvn -v (Apache Maven): 3.6.3
Python --version: 3.10.10

### CODE REVIEW

File: docs/start-here.md [hunk 1]
```diff
@@ -328,3 +328,5 @@ If you think this guide can be improved in any way (e.g., you caught a typo or t
 + Results reproduced by [@Edward-J-Xu](https://github.com/Edward-J-Xu) on 2023-09-04 (commit [`6030b1`](https://github.com/castorini/anserini/commit/6030b1886d3b501c69f258f9f716508bf7fb80ff))
 + Results reproduced by [@Sumon-Kanti-Dey](https://github.com/SumonKantiDey) on 2023-09-04 (commit [`a9de13a`](https://github.com/castorini/anserini/commit/a9de13a81194189bbbffa39d294acd59372229a6))
 + Results reproduced by [@MojTabaa4](https://github.com/MojTabaa4) on 2023-09-13 (commit [`adf480b`](https://github.com/castorini/anserini/commit/ae6ff62f1830f8e3f38199a7d8f723940a62fb28))
++ Results reproduced by [@TheFJcurve](https://github.com/TheFJcurve) on 2023-09-19 (commit c033f71b4d426ccce0488df464784b1233970e1d)
```
Comment (lintool) [inline docs/start-here.md:331]: change commit id to actual URL please?

**Comment by lintool** [comment]
And please continue along the onboarding path!

**Comment by TheFJcurve** [comment]
Hello professor, I am terribly sorry; I am not well versed with the technical know-hows of GitHub. Can you let me know or direct me to a site that will tell me how to do the same? I would be grateful. Thanking you,Sargun From: Jimmy Lin ***@***.***>Date: Monday, September 18, 2023 at 7:08 PMTo: castorini/anserini ***@***.***>Cc: Sargun ***@***.***>, Author ***@***.***>Subject: Re: [castorini/anserini] Adding to Reproduction Log (#2197) (PR ***@***.*** commented on this pull request.In docs/start-here.md:> @@ -328,3 +328,5 @@ If you think this guide can be improved in any way (e.g., you caught a typo or t + Results reproduced by ***@***.***(https://github.com/Edward-J-Xu) on 2023-09-04 (commit [`6030b1`](https://github.com/castorini/anserini/commit/6030b1886d3b501c69f258f9f716508bf7fb80ff)) + Results reproduced by ***@***.***(https://github.com/SumonKantiDey) on 2023-09-04 (commit [`a9de13a`](https://github.com/castorini/anserini/commit/a9de13a81194189bbbffa39d294acd59372229a6)) + Results reproduced by ***@***.***(https://github.com/MojTabaa4) on 2023-09-13 (commit [`adf480b`](https://github.com/castorini/anserini/commit/ae6ff62f1830f8e3f38199a7d8f723940a62fb28))++ Results reproduced by ***@***.***(https://github.com/TheFJcurve) on 2023-09-19 (commit c033f71b4d426ccce0488df464784b1233970e1d)change commit id to actual URL please?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by lintool** [comment]
Having seen no follow-up, closing.

**Pull request closed without merging**